modular_citadel/code/modules/client/loadout/__donator.dm

```diff
@@ -551,6 +551,6 @@
 
 /datum/gear/donator/smaiden
 	name = "Shrine Maiden"
```

Comment:
```suggestion
	name = "shrine maiden outfit"
```
'shrine maiden' isn't a name
lack of object definition makes this show up as
>_This is **Mother Fucker**!_
>_She is wearing a **Shrine Maiden**_

as opposed to

>_This is **Mother Fucker**!_
>_She is wearing a **shrine maiden outfit**_

Reply:
notice that I only changed one thing. So... blame Jay for that. I'll fix it up.